css-typed-om/EXPLAINER.md

```diff
@@ -0,0 +1,117 @@
+# CSS Typed OM Explained
+
+CSS Typed OM is the extensable API for the CSSOM that reduces the burden of manipulating a CSS propeerty's value via string manipulation. It does so by exposing CSS values as typed JavaScript objects which help you circumvent the browser's CSS parser.
+
+## Motivation
+
+The benefits of CSS Typed OM are as follows:
+* Allows the performant manipulation of values assigned to CSS properties. 
+* Being able to write more maintainable code:
+	* Your code is easier to understand,
+	* Easier to write more generic code.
+
+### Examples
+
+### Getting a property's value
+
+To get specified and computed values for CSS properties of an element in CSSOM we get them via accessing them of the `.style` attribute on an element and the `getComputedStyle()` object respectively. 
```

Comment:
Maybe somehow indicate that this is the old way?

Reply:
Done